pyvera/subscribe.py

```diff
@@ -123,23 +123,33 @@ def _run_poll_server(self):
                 device_data, timestamp = (
                     controller.get_changed_devices(timestamp))
                 LOG.debug("Poll returned")
+
                 if self._exiting:
                     continue
-                if not device_data:
+
+                if device_data:
+                    self._event(device_data)
+                else:
                     LOG.debug("No changes in poll interval")
-                    continue
-                self._event(device_data)
-                time.sleep(1)
-            except requests.RequestException:
-                LOG.info("Could not contact Vera - will retry in %ss",
-                         SUBSCRIPTION_RETRY)
-                time.sleep(SUBSCRIPTION_RETRY)
-            except json.decoder.JSONDecodeError:
-                LOG.info("Response was malfomed - will retry in %ss",
-                         SUBSCRIPTION_RETRY)
-                time.sleep(SUBSCRIPTION_RETRY)
+
+                continue
+
```

Comment:
Just noticed you deleted my `time.sleep(1)`. 

It was quite a while back - but I think I found a case where it was possible to kill vera by polling too much when there were a lot of events - so I ended up using short timeout for the poss (so in most cases HA responds quickly) - along with this sleep to slow things down during very busy times.

Reply:
Yeah, no problem. HA is very responsive. Moreso than Vera's native UI, so there's obviously goodness in your choices there.